editor.planx.uk/src/pages/FlowEditor/components/Team/TeamMembers.tsx

```diff
@@ -1,19 +1,162 @@
+  teamMembersByRole: Record<string, TeamMember[]>;
 }
 
 export const TeamMembers: React.FC<Props> = ({ teamMembersByRole }) => {
+  const platformAdmins = (teamMembersByRole.platformAdmin || []).filter(
+    (member) => member.email,
+  );
+  const otherRoles = Object.keys(teamMembersByRole)
+    .filter((role) => role !== "platformAdmin")
+    .reduce((acc: TeamMember[], role) => {
+      return acc.concat(teamMembersByRole[role]);
+    }, []);
+
+  const archivedMembers = otherRoles.filter(
+    (member) => member.role !== "platformAdmin" && !member.email,
+  );
+
+  const getRoleLabel = (role: string) => {
+    return roleLabels[role] || role;
+  };
+
+  const renderMemberTable = (members: TeamMember[]) => {
+    if (members.length === 0) {
+      return (
+        <Table>
+          <TableHead>
+            <TableRow>
+              <TableCell>
+                <strong>No members found</strong>
+              </TableCell>
+            </TableRow>
+          </TableHead>
+        </Table>
+      );
+    }
+    return (
+      <TableContainer>
+        <Table>
+          <TableHead>
+            <StyledTableRow>
+              <TableCell sx={{ width: 300 }}>
+                <strong>User</strong>
+              </TableCell>
+              <TableCell sx={{ width: 200 }}>
+                <strong>Role</strong>
+              </TableCell>
+              <TableCell>
+                <strong>Email</strong>
+              </TableCell>
+            </StyledTableRow>
+          </TableHead>
+          <TableBody>
+            {members.map((member) => (
+              <StyledTableRow key={member.id}>
+                <TableCell
+                  sx={{
+                    display: "flex",
+                    flexDirection: "row",
+                    alignItems: "center",
+                  }}
+                >
+                  <StyledAvatar>
+                    {member.firstName[0]}
+                    {member.lastName[0]}
+                  </StyledAvatar>
+                  {member.firstName} {member.lastName}
+                </TableCell>
+                <TableCell>
+                  <Chip
+                    label={getRoleLabel(member.role)}
+                    size="small"
+                    sx={{ background: "#ddd" }}
+                  />
+                </TableCell>
+                <TableCell>{member.email}</TableCell>
+              </StyledTableRow>
+            ))}
+          </TableBody>
+        </Table>
+      </TableContainer>
+    );
+  };
+
   return (
-    <Box component="pre" sx={{ fontSize: 12 }}>
-      {JSON.stringify(teamMembersByRole, null, 4)}
-    </Box>
+    <Container maxWidth="contentWrap">
+      <Box py={7}>
+        <EditorRow>
+          <Typography variant="h2" component="h3" gutterBottom>
+            Team editors
+          </Typography>
+          <Typography variant="body1">
+            Editors have access to edit your services.
+          </Typography>
+          {renderMemberTable(otherRoles)}
```

Comment:
nit: Having a function here is a little bit of an anti-pattern I think. 

For readability, I'd suggest making a React component. Something like this - 

```tsx
<MemberTable members={otherRoles}/>
```

This could live outside of the scope of `TeamMembers` and we'd end up with easier to test components as we can just pass props in. Happy to talk through this refactor if a call is the best way 👍